src/rpc/blockchain.cpp

```diff
@@ -1531,9 +1531,28 @@ UniValue getchaintxstats(const JSONRPCRequest& request)
     return ret;
 }
 
+UniValue dumpmempool(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() > 0)
+        throw std::runtime_error(
+            "dumpmempool\n"
+            "\nDumps to disk the mempool.\n"
```

Comment:
I'm not the one to complain about this: but should it be `"Dumps the mempool to disk"?